app/controllers/site_page_controller.rb

```diff
@@ -81,6 +81,9 @@ def homepage
   def open_content
   end
 
+  def open_content_v2
```

Comment:
I'm not sure if this is needed, but guess it won't hurt to void this in case we need to add stuff later.